src/core/lib/debug/trace_flags.yaml

```diff
@@ -192,27 +192,27 @@ inproc:
 lb_policy_refcount:
   debug_only: true
   default: false
-  description: null
+  description: LB policy refcounting.
 metadata_query:
   default: false
-  description: null
+  description: GCP metadata queries.
 op_failure:
   default: false
-  description: Error information when failure is pushed onto a completion queue.
+  description: Error information when failure is pushed onto a completion queue. API tracing must be enabled for this flag to have any effect.
```

Comment:
Instead of saying "API tracing", maybe say "The 'api' tracer", to make it clearer what this means?

Same thing on lines 215 and 239.